docs/components/plugins/api/APISectionTypes.tsx

```diff
@@ -26,8 +26,10 @@ export type APISectionTypesProps = {
 };
 
 const defineLiteralType = (types: TypeDefinitionData[]): JSX.Element | null => {
-  const uniqueTypes = Array.from(new Set(types.map((t: TypeDefinitionData) => typeof t.value)));
-  if (uniqueTypes.length === 1) {
+  const uniqueTypes = Array.from(
+    new Set(types.map((t: TypeDefinitionData) => t.value && typeof t.value))
+  );
+  if (uniqueTypes.length === 1 && uniqueTypes.filter(Boolean).length === 1) {
```

Comment:
Do you need this first check at all? 🤔
```suggestion
  if (uniqueTypes.filter(Boolean).length === 1) {
```

Reply:
Yes, it's because I want to see output only when array looks like: `['string']`, not when it looks like `['string', undefinded]`.